### PR TITLE
Feature/settings

### PR DESCRIPTION
**Pull request by Singh-Ish**

### CODE REVIEW

File: src/components/Ublock/Ublock.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface UblockProps {
     noBlock?: boolean
     wide?: boolean
     noPadding?: boolean
+    className?: string | undefined
```
Comment (zahraard) [inline src/components/Ublock/Ublock.tsx:19]: Is there a need for undefined as you have question mark there?

File: src/components/Ublock/Ublock.tsx [hunk 2]
```diff
@@ -43,7 +45,7 @@ const Ublock: React.FC<UblockProps> = ({
     return (
         <div
             id={id}
-            className={`${blockClassName} ${colorClassname}  ${wavesClassname} ${width} ${largeClassname} ${alertClassname} ${wideClassName} ${noPaddingClassname}`}
+            className={`${blockClassName} ${className} ${colorClassname}  ${wavesClassname} ${width} ${largeClassname} ${alertClassname} ${wideClassName} ${noPaddingClassname}`}
```
Comment (zahraard) [inline src/components/Ublock/Ublock.tsx:48]: className could be undefined, it's better to create a variable based on it first, like:
```suggestion
          const customClassName = className ? className : ''
```
and then use the variable .

**Review by zahraard — COMMENTED** [review]
Great job! could you please add test and story for the added prop.
Thanks!

File: src/components/Ublock/Ublock.tsx (hunk 1 above)
Comment (Singh-Ish) [reply src/components/Ublock/Ublock.tsx]: removed undefined

**Review by zahraard — APPROVED** [review]
Looks good, Thank you ;)

**Pull request merged**